### PR TITLE
GSG: facelift

### PR DESCRIPTION
**Pull request by carlescufi**
Quick facelift to the Getting Started Guide to make it more readable.

### CODE REVIEW

File: doc/getting_started/index.rst [hunk 1]
```diff
@@ -12,49 +12,21 @@ environment, then build and run a sample application.
...
+**************************
 
 Follow an operating system specific guide, then come back to this page.
 
 .. toctree::
-   :maxdepth: 1
+   :hidden:
 
-   Linux <installation_linux.rst>
-   macOS <installation_mac.rst>
```
Comment (mbolivar) [inline doc/getting_started/index.rst]: I think the toctree looks fine, but ok :)

File: doc/getting_started/installation_linux.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _installation_linux:
 
-Install Linux Host Dependencies
-###############################
+Install the required tools on Linux
```
Comment (mbolivar) [inline doc/getting_started/installation_linux.rst]: This PR is generally replacing section capitalization with uncapitalized text. Why?

Why is this an improvement also? Are users unfamiliar with the host vs target distinction?

**Comment by mbolivar** [comment]
A couple of doc build errors also

**Review by dbkinder — CHANGES_REQUESTED** [review]
Hi Carles, as we talked about in the last TSC meeting where there was a presentation made about developer experience testing, there's an effort underway to do a revamping of the getting started material.  I've been meeting with @nashif and the DX team to go through their recommendations and I'll have a PR submitted next week (I hope) with changes that will begin to address the findings.  That said, I'd recommend closing your PR so we can coordinate these DX improvements.

**Comment by carlescufi** [comment]
@dbkinder very well thanks for the heads-up. Please do bear in mind the changes I proposed here. 

@mbolivar thanks for the review anyway!

**Pull request closed without merging**